package.json

```diff
@@ -24,6 +24,7 @@
     "figlet": "^1.8.0",
     "fs": "^0.0.1-security",
     "lodash": "^4.17.21",
+    "mqdockerup": "file:",
```

Comment:
Do we need to self reference in the package.json here?

Reply:
I don't know, it added itself automatically when I did npm install, maybe it is the explcita version

Reply:
removed